Microsoft.Toolkit.Uwp.UI.Controls/TextToolbar/TextToolbar.Events.cs

```diff
@@ -0,0 +1,317 @@
+                    foreach (DefaultButton item in e.NewItems)
+                    {
+                        RemoveDefaultButton(item);
+                    }
+
+                    break;
+
+                case NotifyCollectionChangedAction.Remove:
+                    var root = GetTemplateChild(RootControl) as CommandBar;
+                    if (root != null)
+                    {
+                        foreach (DefaultButton item in e.OldItems)
+                        {
+                            AddToolbarItem(item.Button, root);
+                        }
+                    }
+
+                    break;
+            }
+        }
+
+        /// <summary>
+        /// Ensures that the Toolbar updates the position of a Toolbar Item if it's position changes.
+        /// </summary>
+        /// <param name="sender">Toolbar Button</param>
+        /// <param name="e">Property Changed Event</param>
+        private void ToolbarItemPropertyChanged(object sender, System.ComponentModel.PropertyChangedEventArgs e)
+        {
+            var root = GetTemplateChild(RootControl) as CommandBar;
+            if (root != null)
+            {
+                if (e.PropertyName == nameof(IToolbarItem.Position))
+                {
+                    MoveToolbarItem(sender as IToolbarItem, root);
+                }
+            }
+        }
+
+        /// <summary>
+        /// Checks if a Shortcut Combination is pressed, by going through the list of attached buttons.
+        /// </summary>
+        /// <param name="sender">RichEditBox</param>
+        /// <param name="e">Key args</param>
+        private void Editor_KeyDown(object sender, KeyRoutedEventArgs e)
+        {
+            LastKeyPress = e.Key;
+
+            var root = GetTemplateChild(RootControl) as CommandBar;
+            if (root != null)
+            {
+                if (ControlKeyDown && e.Key != VirtualKey.Control)
+                {
+                    var key = FindBestAlternativeKey(e.Key);
+
+                    var matchingButtons = root.PrimaryCommands.OfType<ToolbarButton>().Where(item => item.ShortcutKey == key);
+                    if (matchingButtons.Any() && e.Handled)
+                    {
+                        Editor.Document.Undo();
+                        if (string.IsNullOrWhiteSpace(Editor.Document.Selection.Text))
+                        {
+                            Editor.Document.Redo();
+                        }
+                    }
+
+                    var args = new ShortcutKeyRequestArgs(key, ShiftKeyDown, e);
+                    foreach (var button in matchingButtons)
+                    {
+                        if (button != null && !args.Handled)
+                        {
+                            button.ShortcutRequested(ref args);
+                        }
+                    }
+
+                    ShortcutRequested?.Invoke(this, args);
+                    e.Handled = args.Handled;
```

Comment:
What it does is it finds any button that has a key that matches the CTRL shortcut, if it overrides an Editor Keydown, then it undoes the last action (as the keydown event might have changed something), it then checks if there is a selection after the undo, if there isn't, then nothing has changed, so redo the last action. This is just for overriding CTRL + - at the moment, as it seems to erase the last word by default. 
If a CTRL key wasn't overridden, it won't do anything.